decisions/wc_offmap_decisions.txt

```diff
@@ -616,7 +724,138 @@ offmap_decisions = {
 					}
 				}
 			}
-			FROM = { hidden_effect = { set_character_flag = requested_offmap_power } }
+			FROM = {
+				hidden_effect = {
+					set_character_flag = requested_offmap_power
+					character_event = { id = WCVOD.100 }
+					random_list = {
+						5 = {
+							character_event = { id = WCVOD.300 }
+							modifier = {
+								factor = 4
+								check_variable = { which = void_corruption value = 77 }
+							}
+							modifier = {
+								factor = 2
+								check_variable = { which = void_corruption value = 88 }
+							}
+							modifier = {
+								factor = 2
+								check_variable = { which = void_corruption value = 99 }
+							}
+						}
+						100 = {
+							modifier = {
+								factor = 0.8
+								check_variable = { which = void_corruption value = 77 }
+							}
+							modifier = {
+								factor = 0.5
+								check_variable = { which = void_corruption value = 88 }
+							}
+							modifier = {
+								factor = 0.25
+								check_variable = { which = void_corruption value = 99 }
+							}
+						}
+					}
+				}
```

Comment:
That's not how corruption should be done. There are effects in `wc_corruption_scripted_effects.txt`. `WCVOD.100` is used for a passive increase if you follow Void religion or you're Shadow Priest.